nanome/_internal/_structure/_io/_pdb/structure.py

```diff
@@ -57,6 +58,18 @@ def structure_molecule(atoms, compnds):
                 all_chains[chain_id]._add_residue(residue)
             all_residues[residue_id]._add_atom(atom)
             all_atoms[atom_id] = atom
+        conect_lookup[atom._serial] = atom
+    for conect_list in conects.values():
+        serials = conect_list[-1].atoms_serial_numbers
```

Comment:
Am curious, why last CONECT record instead of first? Shouldn't they all have the same `atom_serial_numbers`?